controllers/hivelocitymachine_controller_test.go

```diff
@@ -148,7 +148,11 @@ var _ = Describe("HivelocityMachineReconciler", func() {
 				},
 				Spec: infrav1.HivelocityMachineSpec{
 					ImageName: "Ubuntu 20.x",
-					Type:      "hvCustom",
+					DeviceSelector: infrav1.DeviceSelector{
+						MatchLabels: map[string]string{
+							"caphvlabel:caphv-device-type": "hvCustom",
```

Comment:
there shouldn't be this prefix caphvlabel.. The user should just specify "caphv-device-type: hvCustom"